apparmor.d/groups/bus/ibus-dconf

```diff
@@ -33,16 +33,16 @@ profile ibus-dconf @{exec_path} flags=(attach_disconnected) {
   /etc/dconf/db/ibus r,
   /etc/dconf/profile/ibus r,
 
-  /var/lib/gdm{3,}/.config/ibus/bus/{,@{hex}-unix-wayland-[0-9]*} r,
```

Comment:
Can we avoid such constructs? It's cryptic even for a professional, let alone novice.